src/Service/Resolvers/SqlMutationEngine.cs

```diff
@@ -825,5 +919,37 @@ private HttpContext GetHttpContext()
         {
             return _httpContextAccessor.HttpContext!;
         }
+
+        /// <summary>
+        /// Helper method to construct a transactionscope object at Read Committed isolation level
```

Comment:
>Helper method to construct a transactionscope object at Read Committed isolation level [](http://example.com/codeflow?start=12&length=86)

Can you add more documentation either here in comments and the PR description which show the decision making in choosing read committed versus serializable isolation levels?

Reply:
The logic is updated to select the isolation level based on the database type. The isolation level chosen is the default isolation level for each DB type.  I've updated the comments with the same information